app/src/main/java/ju/video/player/view/videoconvert/VideoConvertFrame.java

```diff
@@ -1,94 +1,177 @@
+/**
+ * Copyright (c) 2022 Jala University.
+ *
+ * This software is the confidential and proprieraty information of Jalasoft
+ * ("Confidential Information"). You shall not disclose such Confidential
+ * Information and shall use it only in accordance with the terms of the
+ * Licence agreement you entered into with Jalasoft
+ */
 package ju.video.player.view.videoconvert;
 
-import javax.swing.BorderFactory;
 import javax.swing.JPanel;
-import javax.swing.border.Border;
-
 import ju.video.player.controller.ConvertController;
-import ju.video.player.view.MainFrame;
+import ju.video.player.controller.FormatConvertController;
 import ju.video.player.view.commons.UIColor;
-import ju.video.player.view.information.search.FolderChooserButton;
-import ju.video.player.view.materialDesing.ResponsiveSwingMaterialDesign;
-import ju.video.player.view.materialDesing.components.date.fields.DateField;
-import ju.video.player.view.materialDesing.components.image.ImageLabel;
-import ju.video.player.view.materialDesing.components.utils.ImageRoundedBorder;
 import ju.video.player.view.materialDesing.components.utils.RoundedBorder;
 import ju.video.player.view.materialDesing.display.FrameUtility;
-import ju.video.player.view.materialDesing.layouts.ResponsiveLayout;
-import java.awt.event.ActionListener;
-import java.awt.Dimension;
 
+import javax.swing.ImageIcon;
 import javax.swing.JButton;
+import javax.swing.JComboBox;
+import javax.swing.JFrame;
+import javax.swing.JLabel;
+import java.awt.Font;
+import java.awt.Color;
+import java.awt.Component;
+import java.awt.FlowLayout;
+
+/**
+ * This class is responsible for running the view to convert.
+ *
+ * @author Jose Antonio Romay Ayarachi 
+ * @version 1.0
+ */
+
+public class VideoConvertFrame extends JFrame {
+
+    private static final int WIDTH_FRAME = 850;
+    private static final int HEIGHT_FRAME = 600;
+    private static final int WIDTH_PANEL = 260;
+    private static final int HEIGHT_PANEL = 150;
+
+    private static final int POSX_BUTTON = 280;
+    private static final int POSY_BUTTON = 460;
+    private static final int WIDTH_BUTTON = 260;
+    private static final int HEIGHT_BUTTON = 50;
+    private static final int RADIUS_BUTTON = 20;
 
-import javax.swing.*;
-import java.awt.*;
-
-import ju.video.player.view.materialDesing.components.input.TextArea;
-import ju.video.player.view.materialDesing.constants.Constant;
-import ju.video.player.view.playlist.playlistpanel.VideoListPanel;
-
-public class VideoConvertFrame extends JFrame{
-    // frame
-    static JFrame f;
-
-    // label
-    static JLabel l, l1, l3, l4;
-
-    
-    // combobox
-    static JComboBox c1, c2;
     private ConvertController controller;
+    private FormatConvertController formatController;
+    private String path;
+
+    private JComboBox<String> combotype;
+    private JComboBox<String> comboformat;
```

Comment:
The name of the variable must respect the established conventions

Reply:
the variables have already been renamed